src/sys/node/public.ts

```diff
@@ -3,19 +3,35 @@ import type { CompilerSystem, Logger } from '@stencil/core/internal';
 /**
  * Creates a "logger", based off of NodeJS APIs, that will be used by the compiler and dev-server.
  * The NodeJS "process" object must be provided as a property in the first argument's object.
- * @param c an object containing a `Process` for Stencil to use
  * @returns a {@link Logger} object
  */
+export declare function createNodeLogger(): Logger;
+
+// TODO: Remove in Stencil v5, use `createNodeLogger` without parameters instead
```

Comment:
Can we tie this to a ticket in Jira? Ideally, this should be formatted in an easily grep-able format:
```suggestion
// TODO(STENCIL-123): Remove in Stencil v5, use `createNodeLogger` without parameters instead
```
Otherwise, we end up spending time trying to marry this TODO to something we may/may not have in Jira already

Reply:
created `STENCIL-1196`